apps/academy-wep/components/CommunityStats.tsx

```
@@ -0,0 +1,50 @@
+		<div className="relative flex justify-center md:justify-end">
+			<div className="absolute inset-0 bg-gradient-to-br from-[#7CC635]/20 to-blue-500/20 rounded-full blur-xl transform rotate-6 scale-95"></div>
+			<div className="relative bg-white p-4 sm:p-6 rounded-full shadow-lg border border-gray-100 w-36 h-36 sm:w-40 sm:h-40 md:w-44 md:h-44 lg:w-48 lg:h-48 xl:w-52 xl:h-52 flex items-center justify-center">
+				<div className="relative z-10 flex flex-col items-center">
+					<svg
+						width="64"
+						height="64"
+						viewBox="0 0 32 32"
+						fill="none"
+						xmlns="http://www.w3.org/2000/svg"
+						className="mb-2 sm:mb-3 md:mb-4"
+					>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)